libs/common-components/src/lib/zoom.directive.ts

```diff
@@ -34,17 +30,15 @@ export class ZoomDirective implements OnInit, OnDestroy, AfterViewInit {
   }
 
   private setStyles(zoom) {
-    if (! this.zoomActive) {
-      return;
```

Comment:
i think you shouldn't delete this check, setStyles calls from callback, it is better to remove the same check in ngOnInit